libsolutil/CMakeLists.txt

```diff
@@ -1,44 +1,46 @@
 set(sources
-	Algorithms.h
-	AnsiColorized.h
-	Assertions.h
-	Common.h
-	CommonData.cpp
-	CommonData.h
-	CommonIO.cpp
-	CommonIO.h
-	cxx20.h
-	Exceptions.cpp
-	Exceptions.h
-	ErrorCodes.h
-	FixedHash.h
-	FunctionSelector.h
-	IndentedWriter.cpp
-	IndentedWriter.h
-	IpfsHash.cpp
-	IpfsHash.h
-	JSON.cpp
-	JSON.h
-	Keccak256.cpp
-	Keccak256.h
-	LazyInit.h
-	LEB128.h
-	Numeric.cpp
-	Numeric.h
-	picosha2.h
-	Result.h
-	SetOnce.h
-	StringUtils.cpp
-	StringUtils.h
-	SwarmHash.cpp
-	SwarmHash.h
-	UTF8.cpp
-	UTF8.h
-	vector_ref.h
-	Views.h
-	Visitor.h
-	Whiskers.cpp
-	Whiskers.h
+        Algorithms.h
+        AnsiColorized.h
+        Assertions.h
+        Common.h
```

Comment:
This looks like the whole list was rewritten because indentation has been changed here from spaces to tabs. Please keep the original style.